File: source/aoBlogs2/Models/BlogEntryCtaRules.vb

Imports Contensive.Models.Db

Namespace Models
    Public Class BlogEntryCTARuleModel
        Inherits DbBaseModel
        '
        '====================================================================================================
        ''' <summary>
        '''table definition
        '''</summary>
        Public Shared ReadOnly Property tableMetadata As DbBaseTableMetadataModel = New DbBaseTableMetadataModel("Blog Entry CTA Rules", "ccBlogEntryCTARules", "default", False)
        '
        '====================================================================================================
        ' -- instance properties
        Public Property blogentryid As Integer
        Public Property calltoactionid As Integer
    End Class
End Namespace

File: source/aoBlogs2/Models/Db/BlogEntryCtaRules.cs

using Contensive.Models.Db;

namespace Contensive.Blog.Models {
    public class BlogEntryCTARuleModel : Contensive.Models.Db.DbBaseModel {
        // 
        // ====================================================================================================
        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Blog Entry CTA Rules", "BlogEntryCTARules", "default", false);
        // 
        // ====================================================================================================
        // -- instance properties
        public int blogentryid { get; set; }
        public int calltoactionid { get; set; }
    }
}